### PR TITLE
Fixed overlapping of TOC accordion

### PR DESCRIPTION
**Pull request by SteveSamJacob19**
Also changed the z-index issue of the toc accordion by changing its z-index to 10000 as the copy_confirm button has z-index of 9997

### CODE REVIEW

File: src/main/content/_assets/js/openliberty.js [hunk 1]
```diff
@@ -70,18 +70,25 @@ var openliberty = (function() {
...
         // push toc column, toc indicator and code column down below nav bar
-        $("#toc_column").css("top", nav_height + "px");
+        $("#toc_column").css("top",accord_height + nav_height + "px");
         $("#toc_indicator").css("margin-top", nav_height + "px");
 
         // add margin-top to body so page doesn't jump when nav slides into view
         $('body').css("margin-top", nav_height + "px");
 
+        // To set the toc column to the top when mobile toc accordion returns to original position
+        var mobile = document.getElementById('mobile_toc_accordion_container');
```
Comment (natalie-bernhard) [inline src/main/content/_assets/js/openliberty.js]: Can you make line 86 also Jquery please?

File: src/main/content/_assets/js/openliberty.js [hunk 2]
```diff
@@ -70,18 +70,25 @@ var openliberty = (function() {
...
+        if(!mobile.classList.contains("fixed_toc_accordion")){
+            console.log("class found");
```
Comment (natalie-bernhard) [inline src/main/content/_assets/js/openliberty.js]: please remove this console.log

File: src/main/content/_assets/js/openliberty.js [hunk 3]
```diff
@@ -117,6 +125,13 @@ var openliberty = (function() {
         // fix mobile and tablet toc accordion to top of screen again
         $("#tablet_toc_accordion_container").css("top", "0px");
 
+        // to set toc column below the mobile toc accordion only when nav bar is hidden
+        var mobile = document.getElementById('mobile_toc_accordion_container');
```
Comment (natalie-bernhard) [inline src/main/content/_assets/js/openliberty.js]: Please change line 129 to use Jquery

**Review by natalie-bernhard — CHANGES_REQUESTED** [review]
Once you have addressed the other comments, please rebase this branch onto draft

File: src/main/content/_assets/js/openliberty.js [hunk 4]
```diff
@@ -117,6 +124,13 @@ var openliberty = (function() {
         // fix mobile and tablet toc accordion to top of screen again
         $("#tablet_toc_accordion_container").css("top", "0px");
 
+        // to set toc column below the mobile toc accordion only when nav bar is hidden
+        var mobile = $('#mobile_toc_accordion_container');
+        if(mobile.classList.contains("fixed_toc_accordion")){
+            console.log("class found");
```
Comment (natalie-bernhard) [inline src/main/content/_assets/js/openliberty.js]: Please remove this console.log

**Review by natalie-bernhard — CHANGES_REQUESTED** [review]

**Review by steven1046 — APPROVED** [review]

**Review by natalie-bernhard — APPROVED** [review]

**Pull request merged**